content/Teams/stemtrust/index.md

```
@@ -0,0 +1,123 @@
+
+## 👥 Team Members
+
+- **Adetunji Akeem** — Team Lead & Software Developer
+- **Salaudeen Muhammad Mustapha** — Developer & Project Tester
+- **Benshak Tom** — Project Manager
+
+---
+
+## Pages
+
+## Pages
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)